server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -74,8 +74,11 @@ protected DivTag renderInputTags(
         FieldWithLabel.input()
             .setFieldName(addressQuestion.getStatePath().toString())
             .setLabelText(messages.at(MessageKey.ADDRESS_LABEL_STATE.getKeyName()))
+            .setValue(
+                addressQuestion
+                    .getDefaultState()
+                    .orElse(addressQuestion.getStateValue().orElse("")))
```

Comment:
Shouldn't be the other way around? 

`addressQuestion.getStateValue().orElse(adressQuestion.getDefaultState().orElse(""))`

Reply:
No we should use Default state first. The reason being applicants are using- WA, Wash, Washington, Wa$hingt0n many cretive ways to fill the form. We should first check of the admin has set a default value, if yes, use it.. or not, use the value in the system (which is usually empty)

Reply:
Doesn't it mean that if I (applicant) selected "California", saved the answer and then came back to it - wouldn't I see "Washington" instead of "California"? Essentially your change will *always* show default state even if applicant answered a different state?

Reply:
Yes that is exactly the expected behavior. If the admin has set that this program is only for Washington residents and wants to address to be in-state, than, the the state must by Washington. They cannot be out state addresses. This will also help reduce noise in the admin side.

Reply:
That's not how I thought "default state" works at all. I thought that "default state" is just that - default. With an option to change it. If we want to set state to be unchangeable (always be "Washington") - then we should make the State field uneditable. So that user can't change it at all. That should be quite simple change - set it to readonly: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly

Additionally we should rename the field "Default state" in admin question page to something like "Force state" or something similar.